opcache/reset.yml

```diff
@@ -0,0 +1,6 @@
+tasks:
+  opcache:reset:
+    desc: 'Php OPCache Reset'
+    script:
+      - curl -k https://localhost/opcache-clear.php
```

Comment:
The URL should be configurable as this specific URL won't work for most of the sites that we maintain.

Additionally, most of our sites are (or at least should be) designed such that new deployments don't reuse the existing opcache, making this unnecessary for most sites.

Given that most sites don't need this and that the command is simple and may vary significantly between sites, perhaps this should be left out of our recipes? Devs can then easily add this if/where/when needed.